### DISCUSSION TITLE
This project is shit

### DISCUSSION THREAD

**Discussion by mabellegonzalesit** [dubinc/dub#460] [General]
Documentation is shit
Product is shit
Signing up is shit
You are a useless piece of shit Steven

**Comment by coderj001** [comment]
> Documentation is shit
> Product is shit
> Signing up is shit
> You are a useless piece of shit Steven

Code is very good though, it would better have docs for local deployment so we could do some test and stuff🧐

**Comment by morphinestyle** [comment]
Unfortunately the documentation is completely missing, and for this reason I will have to opt for another software. However, I don't think it's right to offend anyone or denigrate the project, it's also useless.

**Comment by steven-tey** [comment]
Oops just seeing this – we added a self-hosting guide a while ago that should help with some of these issues: https://dub.co/docs/self-hosting/guide

We're also planning to add a separate local development guide soon – stay tuned!
